instrumentation/netty/netty-3.8/src/latestDepTest/groovy/Netty38ClientTest.groovy

```diff
@@ -97,17 +98,19 @@ class Netty38ClientTest extends HttpClientTest {
           operationName "CONNECT"
           childOf span(0)
           errored true
-          attributes {
-            Class errorClass = ConnectException
-            try {
-              errorClass = Class.forName('io.netty.channel.AbstractChannel$AnnotatedConnectException')
-            } catch (ClassNotFoundException e) {
-              // Older versions use 'java.net.ConnectException' and do not have 'io.netty.channel.AbstractChannel$AnnotatedConnectException'
+          Class errorClass = ConnectException
+          try {
+            errorClass = Class.forName('io.netty.channel.AbstractChannel$AnnotatedConnectException')
+          } catch (ClassNotFoundException e) {
+            // Older versions use 'java.net.ConnectException' and do not have 'io.netty.channel.AbstractChannel$AnnotatedConnectException'
+          }
+          event(0) {
+            eventName(SemanticAttributes.EXCEPTION_EVENT_NAME)
+            attributes {
+              "${SemanticAttributes.EXCEPTION_TYPE.key()}" errorClass.name
+              "${SemanticAttributes.EXCEPTION_MESSAGE.key()}" ~/Connection refused:( no further information:)? \/127.0.0.1:$UNUSABLE_PORT/
+              "${SemanticAttributes.EXCEPTION_STACKTRACE.key()}" String
```

Comment:
there are a few places that look like they could use `errorEvent`

Reply:
This one's because the message is matched using regex. I could make errorevent more powerful but didn't find enough of these places to warrant it yet.

Reply:
it looks like you are using `errorEvent` in other places with regex?

Reply:
Ah didn't even notice - `message` is the only one in my function without a type parameter mostly by accident :D Yeah I'll use it then